app/domain/errors.rb

```diff
@@ -31,6 +31,11 @@ module Conjur
       code: "CONJ00074E"
     )
 
+    MissingSecretValue = ::Util::TrackableErrorClass.new(
+      msg:  "Variable {0-variable-id} is empty or not found.",
```

Comment:
@shulifink  Is this error approved? @InbalZilberman @orenbm Don't we want the account?

Reply:
@liavyona @shulifink confirmed she is fine with the message in the issue, if you remember. fire away!

Reply:
fine